woocommerce/woocommerce-ios
# Simplified login: Update store picker with entry point to store creation

Closes: #7902 


### Description

This PR adds updates to the store picker for the simplified login project:
- Optional borders in the account header of the store picker
- New button "Add a Store" as the footer of the store list
- Action sheet for "Create a new store" and "Connect an existing store".
- Semi-bold font for the message on the empty store picker along with more spacing between the image and the text.
- "Add a Store" button at the bottom of the empty store picker.
- Tracks for taps on the new buttons. Also replaced event `site_picker_enter_store_address_tapped` with `site_picker_connect_existing_store_tapped`.

### Testing instructions

With the `.simplifiedLoginFlowI1`  and `.storeCreationMVP` feature flags enabled:
1. Log out of the app if necessary.
2. On the prologue screen, select "Log In"
3. Log in with a valid WP.com email address.
4. Make sure that the store picker look matches the Figma design here: ERGtCFLyiC9CFpHsw1XIid-fi-392%3A3599.
5. Tap "Add a Store" button at the bottom of the screen if the store list is empty, or at the footer of the list. Notice in Xcode console: `🔵 Tracked site_picker_add_a_store_tapped`
6. On the action sheet that appears, tap "Create a new store", notice in Xcode console: `🔵 Tracked site_picker_create_store_tapped`.
7. Dismiss the store creation flow, tap the "Connect existing store" button from the action sheet, notice in Xcode console: `🔵 Tracked site_picker_connect_existing_store_tapped`.

With `.simplifiedLoginFlowI1`  enabled and `.storeCreationMVP` disabled:
- Repeat steps 1-5 in the last case. Notice that when the "Add a Store" button is tapped, you are navigated directly to the site discovery flow.

With `simplifiedLoginFlowI1` disabled: the store picker should look and work exactly as before. The only difference is that tapping on the "Enter Your Store Address" will track `site_picker_connect_existing_store_tapped`.

### Screenshots

<img src="https://user-images.githubusercontent.com/5533851/198543778-b8e84a98-8020-4a95-81df-0a3e144562ad.png" width=320 />
<img src="https://user-images.githubusercontent.com/5533851/198543784-40cff2d6-5003-4f9a-bab0-982bc17179a0.png" width=320 />
<img src="https://user-images.githubusercontent.com/5533851/198543789-263445f9-4a85-4b21-856a-b71575d4b853.png" width=320 />


---
